web/src/styles/app/base.css

```diff
@@ -4,10 +4,6 @@ html {
   font-size: var(--base-font-size);
 }
 
-body {
-  font-family: var(--base-font-family);
-}
-
```

Comment:
@jenniferthibault SOURCE SANS PRO HAS BEEN SMITED FROM THE CODEBASE
![miss you comrade](https://media.giphy.com/media/rHR8qP1mC5V3G/giphy.gif)

Reply:
![thumbs up sheep](https://media.giphy.com/media/tIeCLkB8geYtW/giphy.gif)